### ISSUE TITLE
Extract CORSHandler in library

### ISSUE THREAD

**Issue by hadrienk** [statisticsnorway/data-collector-server-base#1]
I see this class i the same as the one in lds-core

https://github.com/statisticsnorway/data-collector-server-base/blob/3d68617de391321b6b1191bff901e16ce5c7465f/src/main/java/no/ssb/dc/application/controller/CORSHandler.java#L20-23

https://github.com/statisticsnorway/linked-data-store-core/blob/8df5abf197d1fd2daa7b6c60f0d58ef595dbe34b/src/main/java/no/ssb/lds/core/controller/CORSHandler.java#L20-L23

Maybe we could extract the undertow handler we use accross the projects?

**Comment by oranheim** [comment]
Yes, that would be a good thing. Let's review after I have refactored to the whole server base and service instance factories.
